content/en/community/hackathons/usoc23/baby-steps/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Session 02 - Baby Steps
+linkTitle: Bulding Unikernels
+weight: 5
+summary: |
+  We start presenting the modular architecture of Unikraft and how students can build their unikernels using the manual, `make`-based approach.
+  Expected time: 100 minutes
+---
+
+{{< readfile file="/community/hackathons/sessions/setup/content/intro.md" markdown="true" >}}
+
+
+## Reminder
+In the previous [session](community/hackathons/usoc23/overview), we have experimented with building and running several unikernels using instructions from the `README.md` files of the applications.
+Today, we will explore the modular architecture of Unikraft, and how we can leverage that in order to setup, configure, build and run our unikernels from scratch, without the use of scripts.
+
+## Intro
+{{< readfile file="/community/hackathons/sessions/baby-steps/content/intro.md" markdown="true" >}}
```

Comment:
Remove `or use automated [scripts](https://github.com/unikraft-upb/scripts) for a more simplified approach` from the baby-steps `intro.md` file that is read here.